SIL.BuildTasks.AWS/SIL.BuildTasks.AWS.csproj

```diff
@@ -1,52 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net461</TargetFrameworks>
+    <TargetFrameworks>net472</TargetFrameworks>
```

Comment:
Am I correct in assuming that this will not prevent projects that target 4.6.1 from using SIL.BuildTasks?

Reply:
Yes, that's correct. Since this runs in the context of `msbuild`/`dotnet build` it is independent of the target framework. I guess you might get into trouble if you're trying to build on a very old machine that doesn't have .NET Framework >= 4.7 available...